srcpkgs/lsd/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'lsd'
+pkgname=lsd
+version=0.16.0
+revision=1
+build_style=cargo
+hostmakedepends="pkg-config"
+#makedepends="rust cargo"
+depends="awesome-terminal-fonts"
+short_desc="The next gen ls command"
+maintainer="Lugubris <lugubris@disroot.org>"
+license="Apache-2.0"
+homepage="https://github.com/Peltoche/${pkgname}"
+changelog="https://raw.githubusercontent.com/Peltoche/${pkgname}/${version}/CHANGELOG.md"
+distfiles="https://github.com/Peltoche/${pkgname}/archive/${version}.tar.gz"
+checksum=e2406748d78431a1c03bdd2404a204a006c19905d926e41a36587b93a791e003
+
+do_install() {
```

Comment:
why is that function necessary? It seems to mostly replicate the `build_style` function.

Reply:
does the template look good now?